config/locales/en.yml

```diff
@@ -44,3 +44,4 @@ en:
     link_to: "Edit"
   destroy:
     link_to: "Remove"
+    confirm: "Removal is permenant. Are you sure?"
```

Comment:
I think this is spelled "permanent"

Reply:
fixed in dcc959cca01abf4e3155a8da8c80dc0142efc48b